### DISCUSSION TITLE
Can you customize the Nebula theme to have a homepage grid layout, like the default Kadence WP theme layout?

### DISCUSSION THREAD

**Discussion by solkad** [chrisblakley/Nebula#2228] [Q&A] [answered]
Here is what the default Kadence WP theme home page layout looks like. It uses a grid layout.
https://i.ibb.co/hBz8ZwT/Homepage-Kadence-WP.png

The default Nebula home page for the same content has a single column layout, that I think is the standard for Wordpress themes.
https://i.ibb.co/rx5c7zj/Homepage-Nebula.png

Is there a way to switch to a grid layout in Nebula? I looked around in wp-admin>Appearance "Customize" but I couldn't find anything like that.

Is this something that would require custom coding, and if so, would it be difficult?

**Comment by chrisblakley** [comment, marked as answer]
This would be something the Block Editor would need to be used for, or custom code could be used in a custom built template. On a broad spectrum of expertise it wouldn't be the hardest thing to do, but if you are a beginner it may be slightly challenging. That being said, it would also be a great introduction to CSS flexbox or a grid framework.
